DUI3/Speckle.Connectors.DUI/Bindings/OperationProgressManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Concurrent;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Models.Card;
+
+namespace Speckle.Connectors.DUI.Bindings;
+
+/// <summary>
+/// Debouncing progress for every %1 update for UI.
+/// This class requires a specific bridge in its binding, so registering it will create random bridge which we don't want to.
+/// </summary>
+public class OperationProgressManager
```

Comment:
I would make this class a singleton and inject it on usage.  The Bridge isn't needed until set is called so doesn't need to be on the constructor.

How does it know it's done?   Does the diff only care if it's changed, not if it's greater?

Reply:
that's a good point